### PR TITLE
update for 1007, more packaging/war/* fixes

### PR DESCRIPTION
**Pull request by scottmarlow**
Signed-off-by: Scott Marlow <smarlow@redhat.com>

**Fixes Issue**

https://github.com/eclipse-ee4j/jakartaee-tck/issues/988

### CODE REVIEW

**Comment by scottmarlow** [comment]
https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck-scottmarlow/job/ejb30_lite_packaging_morechanges/2/ shows failures still occurring in:

```
com.sun.ts.tests.ejb30.lite.packaging.war.enventry.JsfClient.injectedIntoClient_from_ejblitejsf
com.sun.ts.tests.ejb30.lite.packaging.war.mbean.interceptor.lifecyclecdi.Client.overrideWithPostConstructBean_from_ejbliteservlet
com.sun.ts.tests.ejb30.lite.packaging.war.mbean.interceptor.lifecyclecdi.Client.overrideWithPostConstructBean_from_ejbliteservlet2
com.sun.ts.tests.ejb30.lite.packaging.war.mbean.interceptor.lifecyclecdi.Client.overrideWithRegularMethod_from_ejbliteservlet
com.sun.ts.tests.ejb30.lite.packaging.war.mbean.interceptor.lifecyclecdi.Client.overrideWithRegularMethod_from_ejbliteservlet2
com.sun.ts.tests.ejb30.lite.packaging.war.webinflibonly.Client.beanClassToClientLookup_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.packaging.war.webinflibonly.Client.beanClassToClientLookup_from_ejbliteservlet
com.sun.ts.tests.ejb30.lite.packaging.war.webinflibonly.Client.beanClassToClientLookup_from_ejbliteservlet2
com.sun.ts.tests.ejb30.lite.packaging.war.webinflibonly.Client.clientToBeanClassLookup_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.packaging.war.webinflibonly.Client.clientToBeanClassLookup_from_ejbliteservlet
com.sun.ts.tests.ejb30.lite.packaging.war.webinflibonly.Client.clientToBeanClassLookup_from_ejbliteservlet2
com.sun.ts.tests.ejb30.lite.packaging.war.webinflibonly.Client.crossEJBLookup_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.packaging.war.webinflibonly.Client.crossEJBLookup_from_ejbliteservlet
com.sun.ts.tests.ejb30.lite.packaging.war.webinflibonly.Client.crossEJBLookup_from_ejbliteservlet2
com.sun.ts.tests.ejb30.lite.packaging.war.webinflibonly.JsfClient.beanClassToClientLookup_from_ejblitejsf
com.sun.ts.tests.ejb30.lite.packaging.war.webinflibonly.JsfClient.clientToBeanClassLookup_from_ejblitejsf
com.sun.ts.tests.ejb30.lite.packaging.war.webinflibonly.JsfClient.crossEJBLookup_from_ejblitejsf
com.sun.ts.tests.ejb30.lite.packaging.war.xmloverride.ejbref.Client.availableInEjb_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.packaging.war.xmloverride.ejbref.Client.availableInEjb_from_ejbliteservlet
com.sun.ts.tests.ejb30.lite.packaging.war.xmloverride.ejbref.Client.availableInEjb_from_ejbliteservlet2
com.sun.ts.tests.ejb30.lite.packaging.war.xmloverride.ejbref.Client.resolveByEjbLinkInXml_from_ejblitejsp
com.sun.ts.tests.ejb30.lite.packaging.war.xmloverride.ejbref.Client.resolveByEjbLinkInXml_from_ejbliteservlet
com.sun.ts.tests.ejb30.lite.packaging.war.xmloverride.ejbref.Client.resolveByEjbLinkInXml_from_ejbliteservlet2
com.sun.ts.tests.ejb30.lite.packaging.war.xmloverride.ejbref.JsfClient.availableInEjb_from_ejblitejsf
com.sun.ts.tests.ejb30.lite.packaging.war.xmloverride.ejbref.JsfClient.resolveByEjbLinkInXml_from_ejblitejsf
```

**Review by starksm64 — APPROVED** [review]

**Comment by scottmarlow** [comment]
https://ci.eclipse.org/jakartaee-tck/job/jakartaee-tck-scottmarlow/job/ejb30_lite_packaging_morechanges/14/testReport/ shows all packaging  tests are passing!

File: src/com/sun/ts/tests/ejb30/lite/packaging/war/enventry/ejblitejsf_vehicle_web.xml [hunk 1]
```diff
@@ -34,20 +34,20 @@
     
     <env-entry>
 	<description>spaces should be preserved</description>
-	<env-entry-name>com.sun.ts.tests.ejb30.lite.packaging.war.enventry.Client/myString</env-entry-name>
```
Comment (alwin-joseph) [inline src/com/sun/ts/tests/ejb30/lite/packaging/war/enventry/ejblitejsf_vehicle_web.xml:37]: Maybe this file is not required as it is copied to new folder jsfenventry ?

**Review by alwin-joseph — APPROVED** [review]
LGTM as the tests are passing. 

Please remember to squash merge.

**Pull request merged**